core/src/style/textStyle.cpp

```diff
@@ -87,10 +96,12 @@ void TextStyle::buildPoint(const Point& _point, const DrawRule& _rule, const Pro
     const auto& text = _props.getString(key_name);
     if (text.length() == 0) { return; }
 
-    Label::Options options;
-    options.color = params.fill;
 
-    buffer.addLabel(text, { glm::vec2(_point), glm::vec2(_point) }, Label::Type::point, params, options);
+    if (Tangram::getDebugFlag(Tangram::DebugFlags::labels)) {
+        buffer.addLabel(std::to_string(params.priority), { glm::vec2(_point), glm::vec2(_point) }, Label::Type::debug, params, optionsFromTextParams(params));
```

Comment:
I don't think so, since this needs the feature points/lines etc. Not sure if this would be useful for others to debug though, I was using this only to check if options were good, and too see how many labels with high priority were rejected so this can go away.